src/ophyd_async/tango/demo/_detector.py

```diff
@@ -0,0 +1,46 @@
+import asyncio
+
+from ophyd_async.core import (
+    AsyncStatus,
+    DeviceVector,
+)
+from ophyd_async.tango import TangoReadable
+
+from ._counter import TangoCounter
+from ._mover import TangoMover
+
+
+class TangoDetector(TangoReadable):
```

Comment:
If there is no `trl` passed in `__init__`, does it make sense for this to be a `TangoReadable`? It could be a `StandardReadable` instead and not take `trl` in init

Reply:
I had kept it as a TangoReadable so it could use the device inference methods in TangoDevice. Since we won't be doing that, I will change it to a StandardReadable